.github/workflows/md-lint.yml

```diff
@@ -0,0 +1,41 @@
+name: Lint Docs
+
+on:
+  push:
+    branches-ignore:
+      - master
+      - main
+    paths:
+       - "**/*.md"
+  pull_request:
+    branches:
+      - master
+      - main
+    paths:
+       - "**/*.md"
+
+jobs:
+  build:
+    name: Lint Markdown
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+        
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 'lts/*'
+
+      - name: Install Linter
+        run: npm i --no-save markdownlint-cli2
+
+      - name: Add Matcher
+        uses: xt0rted/markdownlint-problem-matcher@b643b0751c371f357690337d4549221347c0e1bc
```

Comment:
What's this exactly?

Reply:
This provides the inline errors on the `files` tab of a pull request.